ush/templates/FV3LAM_wflow.xml

```diff
@@ -724,7 +724,11 @@ MODULES_RUN_TASK_FP script.
 {%- if do_retro %}
            <and>
              <datadep age="00:00:05:00"><cyclestr>&FG_ROOT;/@Y@m@d@H{{ slash_ensmem_subdir }}/ics/gfs_data.tile7.halo0.nc</cyclestr></datadep>
-             <datadep age="00:00:05:00"><cyclestr offset="-{{ extrn_mdl_ics_offset_hrs }}:00:00">&FG_ROOT;/@Y@m@d@H{{ slash_ensmem_subdir }}/lbcs/gfs_bndy.tile7.{{ "%03d" % boundary_len_hrs }}.nc</cyclestr></datadep>
+             <or>
```

Comment:
@hu5970 Is this the section that you are concerned about? The reason I changed it is that when I initialize the cycle at 00Z and have the ics and lbcs tasks done for 00Z.But because I am having extrn_mdl_ics_offset_hrs=6 for GDASENKF, it can't find lbcs from 18Z and therefore won't start prep_cyc. By changing it to look for lbcs in the past few hours, it will kick off the task since I have lbcs at 00Z.